UnityProject/Assets/Scripts/Shuttles/ShuttleInteract.cs

```diff
@@ -0,0 +1,30 @@
+﻿using PlayGroups.Input;
+using UI;
+using UnityEngine;
+
+public class ShuttleInteract : InputTrigger 
+{
+    public string interactionMessage;
+    public MatrixMove ShuttleMatrixMove;
+    private GameObject player;
+    
+    // Method that brings up shuttle controls.
+    public override void Interact(GameObject originator, Vector3 position, string hand)
+    {
+        player = originator;
+        ControlTabs.ShowShuttleTab(ShuttleMatrixMove);
+    }
+
+    /// <summary>
+    /// Checks if the player has moved away from the shuttle. If the player has moved away it hides the tab.
+    /// </summary>
+    public void Update()
+    {
+        if ( player != null && Vector3.Distance(player.transform.position, transform.position) > 1.5f )
```

Comment:
1) this check is clientside-only in your case, it should be serverside as well (more on that in other comments)
2) It's unnecessary to check for that every frame, I think checking only when player moves would be better
3) use PlayerScript.IsInReach() for range checks instead